### PR TITLE
Update the Address Book topic

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR):
- Adds the methods to change Address from the Address Book on the storefront 
- Removes the wrong "Only EE" label 

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-address-book.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=addres

### CODE REVIEW

File: src/customers/account-dashboard-address-book.md [hunk 1]
```diff
@@ -32,4 +32,16 @@ _[Address Book]({% link configuration/customers/customer-configuration.md %})_
...
+
+## Change an address
+
+1. In the sidebar of the customer account, select **Address Book**.
+
+1. Choose one of the methods to edit:
+
+   - Click the **Change Billing/Shipping Address** in the Default Addresses block.
+
+   - Ckick the **Edit** in the **Additional Address Entries** grid.
```
Comment (dmytro-ch) [inline src/customers/account-dashboard-address-book.md:45]:
```suggestion
   - Click the **Edit** in the **Additional Address Entries** grid.
```

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

File: src/customers/account-dashboard-address-book.md [hunk 2]
```diff
@@ -32,4 +32,16 @@ _[Address Book]({% link configuration/customers/customer-configuration.md %})_
...
+
+1. Make changes necessary, and click <span class="btn">Save Address</span>.
```
Comment (dmytro-ch) [inline src/customers/account-dashboard-address-book.md:47]: I would also recommend adjusting this line a bit.
```suggestion
1. Make the necessary changes, and click <span class="btn">Save Address</span>.
```

**Comment by EvheniiaKolkhiieva** [comment]
Hi @dmytro-ch, thank you for reviewing!

I have implemented your suggestions, please check! ☺️

File: src/customers/account-dashboard-address-book.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-address-book.md]: Done, please check 😊

File: src/customers/account-dashboard-address-book.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-address-book.md]: Done, please check 😊

**Review by dmytro-ch — APPROVED** [review]
@EvheniiaKolkhiieva, great, thank you for the swift update! :)

File: src/customers/account-dashboard-address-book.md [hunk 3]
```diff
@@ -32,4 +32,16 @@ _[Address Book]({% link configuration/customers/customer-configuration.md %})_
...
-_[Additional Address Entries]({% link configuration/customers/customer-configuration.md %})_{:.ee-only}
+_[Additional Address Entries]({% link configuration/customers/customer-configuration.md %})_
+
+## Change an address
+
+1. In the sidebar of the customer account, select **Address Book**.
+
+1. Choose one of the methods to edit:
+
+   - Click the **Change Billing/Shipping Address** in the Default Addresses block.
```
Comment (leslietilling) [inline src/customers/account-dashboard-address-book.md:43]:
```suggestion
   - Click **Change Billing/Shipping Address** in the _Default Addresses_ section.
```

File: src/customers/account-dashboard-address-book.md [hunk 4]
```diff
@@ -32,4 +32,16 @@ _[Address Book]({% link configuration/customers/customer-configuration.md %})_
...
+
+   - Click the **Edit** in the **Additional Address Entries** grid.
```
Comment (leslietilling) [inline src/customers/account-dashboard-address-book.md:45]:
```suggestion
   - Click **Edit** in the _Additional Address Entries_ grid.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for the review :) Done ✅ 
Could you please approve this PR?

Thank you!

File: src/customers/account-dashboard-address-book.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-address-book.md]: Done ✅

File: src/customers/account-dashboard-address-book.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-address-book.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**